packages/server/src/testUtils.ts

```diff
@@ -60,3 +66,33 @@ export function withServer(server) {
     expect(request.result).toStrictEqual(result);
   };
 }
+
+export const generateMockData = async () => {
```

Comment:
This abstraction is sure to bite us in the butt later. How do we extend the mock data without affecting every other test? How do we remember all the different entities being mocked? How do we avoid mocking 4 other things that aren't actually needed for a specific test?

Two proposals:
1. Just create data manually in every single test. This is verbose, but it makes it obvious to the reader what data exists without hopping between files. It also makes it easy to have different data for different tests.
2. Make small, descriptive mock helpers. `const FUNDIES = async()=>CourseModel.create({name:"CS 2500", icalUrl: "fundies.com"})`

we can do both of the above, but in the name of Avoiding Hasty Abstractions, maybe do 1 before 2.

Reply:
That makes sense, I went with #1 for now and we can revisit the abstraction later.